ecc/bls12-377/fp/element_amd64.go

```diff
@@ -7,6 +7,12 @@
 
 package fp
 
+import (
```

Comment:
This works. But couldn't we just do
```
import _ "github.com/consensys/gnark-crypto/field/asm/element_6w"
```
and not import the dummy constant at all?

Reply:
probably yes, will try 👍